test/functional/lua/treesitter_spec.lua

```diff
@@ -292,9 +309,9 @@ static int nlua_schedule(lua_State *const lstate)
     ]]}
 
     exec_lua([[
-      local TSHighlighter = vim.treesitter.TSHighlighter
+      local TSHighlighter = vim.treesitter.highlighter
```

Comment:
weird to still call this TSHighlighter? I'm not sure.

Reply:
Renamed